guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -0,0 +1,121 @@
+[id="configuring-the-active-directory-authentication-source-on-projectserver_{context}"]
+= Configuring the Active Directory authentication source on {ProjectServer}
+
+Enable Active Directory (AD) users to access {Project} by configuring the corresponding authentication provider on your {ProjectServer}.
+
+.Prerequisites
+* The base system of your {ProjectServer} must be joined to an Active Directory (AD) domain.
+To enable AD users to sign in with Kerberos single sign-on, join the system by using the System Security Services Daemon (SSSD) and Samba services.
```

Comment:
What does "the system" refer to? Considering that the command uses `foreman-maintain`, I assume it's either Foreman Server, or, more generic, Smart Proxies?

Reply:
It refers to "the base system of your {ProjectServer}" from the previous line. This part really talks about connecting the underlying system of the Foreman server, not about connecting the Foreman server itself. (Because the part about connecting the Foreman server comes later with `foreman-installer --foreman-ipa-authentication=true`.) Do you think it works this way? Or would you still change the wording @maximiliankolb?

Reply:
@asteflova I still lean towards `{ProjectServer}` to emphasis that it's a task done on Foreman server, not your local host or any registered host. But I am also OK with "base system". Your call.

Reply:
I reworded the line to help emphasize the point. I hope it helps, if not, we can always revisit :)